masterlist.yaml

```
@@ -3680,6 +3680,29 @@ plugins:
         itm: 64
         udr: 11
 
+  - name: 'BostonLibrarySettlement(PRP|Robots|SS2)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/68293/'
+        name: 'Boston Public Library Settlement'
+    group: *preVisCombinedGroup
+    after:
+      - 'PRP.esp'
```

Single-entry lists are written in YAML's flow style:

```suggestion
    after: [ 'PRP.esp' ]
```